tools/test-npm.sh

```diff
@@ -5,13 +5,26 @@ set -e
 # always change the working directory to the project's root directory
 cd $(dirname $0)/..
 
-# pass NODE_EXE from something like Makefile
-# it should point to either {./}node or {./}node.exe, depending on the platform
-if [ -z $NODE_EXE ]; then
+# pass a $NODE environment variable from something like Makefile
+# it should point to either ./{node} or ./{node}.exe, depending on the platform
```

Comment:
What's the significance of the braces in these comments, why not just write `./node`?